orcoursetrion/tests/base.py

```diff
@@ -96,11 +101,16 @@ def callback_team_list(
 
     def callback_team_members(
             self, request, uri, headers,
-            status_code=200, members=TEST_TEAM_MEMBERS
+            status_code=200, members=None
     ):
         """
         Return team membership list
         """
+        # Disabling unused-argument because this is a callback with
+        # required method signature.
+        # pylint: disable=unused-argument,too-many-arguments
```

Comment:
Does this pylint comment actually trickle down to the correct statement (the assert)?